### PR TITLE
xpi filterlist

### PR DESCRIPTION
**Pull request by troysjanda**
Adding XPI FIlterlist I hope that I did this correctly

### CODE REVIEW

File: services/Directory/data/FilterList.json [hunk 1]
```diff
@@ -17342,4 +17342,21 @@
     "licenseId": 8,
     "name": "phishing-filter (Suricata)"
   }
```
Comment (collinbarrett) [inline services/Directory/data/FilterList.json:17344]: might need to add a comma here
```suggestion
  },
```

File: services/Directory/data/FilterList.json [hunk 2]
```diff
@@ -17341,5 +17341,22 @@
...
+
+"id": 2496,
+
+"licenseId": 4,
+
+"name": "XPI's Filter List",
+
+"submissionUrl": "https://github.com/troysjanda/MyBlockLists/issues"
+
+},
```
Comment (collinbarrett) [inline services/Directory/data/FilterList.json:17360]: maybe need to remove this comma at the end. JSON is finnicky. 😀
```suggestion
}
```

**Comment by troysjanda** [comment]
im really new to this very sorry to have causing issues.

**Comment by collinbarrett** [comment]
Haha, no problem at all. Unfortunately, the process to update data is not easy. Hopefully we'll be able to work on #372 eventually.

**Comment by troysjanda** [comment]
Thanks for your patients.

**Pull request merged**